examples/3d/deferred_rendering.rs

```diff
@@ -57,6 +58,7 @@ fn setup(
             specular_map: asset_server.load("environment_maps/pisa_specular_rgb9e5_zstd.ktx2"),
             intensity: 2000.0,
         },
+        MotionBlur::default(),
```

Comment:
I'm not sure why this is here, this scene is too static to show any motion blur effect

Reply:
I added it as a test to make sure it doesn't break across forward/deferred/etc. I can remove though.